mongolog/handlers.py

```diff
@@ -278,6 +227,22 @@ def create_log_record(self, record):
 
         return record
 
+    def formatException(self, ei):
+        """
+        Format and return the specified exception information as a string.
+
+        This default implementation just uses
+        traceback.print_exception()
+        """
+        sio = StringIO()
+
+        traceback.print_exception(ei[0], ei[1], ei[2], None, sio)
+        s = sio.getvalue()
+        sio.close()
+        if s[-1:] == "\n":
```

Comment:
Perhaps use [`os.linesep`](https://docs.python.org/3/library/os.html#os.linesep) for those poor souls on windows